app/models/concerns/oregon_digital/generic_metadata.rb

```diff
@@ -574,12 +571,14 @@ module GenericMetadata
       end
 
       property :based_near, predicate: ::RDF::Vocab::FOAF.based_near, class_name: Hyrax::ControlledVocabularies::Location
+      property :format, predicate: ::RDF::Vocab::DC.format, class_name: OregonDigital::ControlledVocabularies::MediaType
```

Comment:
Why was the `index.as` line removed from :format ?

Reply:
I removed it because I was following the way hyrax had done it previously. https://github.com/samvera/hyrax/blob/5a9d1be16ee1a9150646384471992b03aab527a5/app/models/concerns/hyrax/basic_metadata.rb#L29 I can try and add it back and see how it effects the system. I imagine it wont, so it should be cool.